nestris_ocr/scan_strat/scan_helpers.py

```diff
@@ -15,6 +18,25 @@
     "das": "BD",
 }
 
+REFERENCE_LEVEL_COLORS = (
+    ((0x4A, 0x32, 0xFF), (0x4A, 0xAF, 0xFE)),
```

Comment:
Weren't these already defined in the colors file?

Reply:
Oops, I forgot to remove them here, They're not needed in this file any more.

Reply:
Alamak. I had removed them. Just forgot to save the file, so it wasn't committed 😑

Reply:
Sadly I think that omission is what made the PR appear as having duplication and let to your comment:

> Looks good but looks like a bit of code duplication

Now that I have committed the file correctly, pls check again :)